rust/projects/file-io/project.md

```diff
@@ -64,17 +94,22 @@ methods:
   Return an error if the value is not read successfully.
 
 When setting a key to a value, `kvs` writes the `set` command to disk in a
-write-ahead log. On startup, the commands in the WAL are re-evaluated and the
-state cached in memory.
+sequential log. On startup, the commands in the log are re-evaluated and the log
+pointer (file offset) of the last command to modify each key recorded in the
+in-memory index.
```

Comment:
I have difficulty in understanding this sentence. I find the sentence structure weird, and I am confused by "the last command" here.

Reply:
Ah, I now understand it. We find the last command which modifies each key, and record the log pointer of it in the in-memory index.

Reply:
Can you try to make the sentence more understandable?

Reply:
Done.